.changes/next-release/Feature-546b77ba-20be-4573-811d-6a6d86b47f00.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Created a command to display Toolkit and environment info. These include OS, VS Code, and Toolkit versions."
```

Comment:
Consider something like the following to both inform and guide:
Added "COMMAND_NAME" Command to show AWS Toolkit versioning details that are useful to include with bug reports.